src/ComboBox/ComboBox.svelte

```diff
@@ -205,7 +205,9 @@
   $: menuId = `menu-${id}`;
   $: comboId = `combo-${id}`;
   $: highlightedId = items[highlightedIndex] ? items[highlightedIndex].id : 0;
-  $: filteredItems = items.filter((item) => shouldFilterItem(item, value));
+  $: /** @type Array<ComboBoxItem> */ filteredItems = items.filter((item) =>
```

Comment:
Shouldn't the type already be inferred correctly based on the type of `items`?

Reply:
I respect that it takes time to write such a helpful reply. I will fix it.